common/xdc/event_generator.go

```diff
@@ -239,7 +304,7 @@ func (g *EventGenerator) getNextPossibleVertex() Vertex {
 		}
 		neighbors := g.connections[leaf]
 		for _, nextV := range neighbors {
-			if nextV.GetCondition() == nil || (nextV.GetCondition() != nil && nextV.GetCondition()(g.previousVertexes)) {
+			if nextV.GetCondition() == nil || (nextV.GetCondition() != nil && nextV.GetCondition()()) {
```

Comment:
this is equivalent to `nextV.GetCondition() == nil ||  nextV.GetCondition()()`

Reply:
on line 312, it it guaranteed to be `isAccessible == false` ?

Reply:
On 312, yes it is guaranteed or it will break the loop